tests/testthat/test-map_migration.R

```diff
@@ -0,0 +1,44 @@
+test_map_migration <- function(df) {
```

Comment:
This is not a test_that function, neither does it run some kind of test. Why is this in the tests folder and called test-map_migration?